src/k6/build.gleam

```diff
@@ -0,0 +1,17 @@
+import esgleam
+import k6/internal
+import simplifile
+
+pub fn main() {
+  let assert Ok(cwd) = simplifile.current_directory()
+  let project_name =
+    cwd
+    |> internal.read_gleam_toml()
+    |> internal.get_project_name()
+
+  esgleam.new("./dist/static")
+  |> esgleam.entry(project_name <> ".gleam")
+  |> esgleam.target("es2020")
+  |> esgleam.raw("--external:k6/http")
```

Comment:
I don't know how to make it configurable but it's ok for now.

Reply:
Oh the external flag can use `*` to match wildcard. Will find the time to try. 😃

Reply:
Ref https://esbuild.github.io/api/#external